.github/labeler.yml

```
@@ -0,0 +1,5 @@
+icons: ['icons/*', 'src/core/icons/*.ts']
+translations: ['package.nls.*.json', 'src/core/i18n']
```

We also have to match the file "package.nls.json" for the English translations.

```suggestion
translations: ['package.nls*.json', 'src/core/i18n']
```

---

Done